BlackMarket/Views/Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Dashboard.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 25/09/2023.
+//
+
+import SwiftUI
+
+struct DashboardView: View {
+  @EnvironmentObject var modelData: ModelData
+  
+  var body: some View {
+    VStack {
+      ScrollView {
+        Spacer().frame(height: 1)
```

Comment:
SwiftUI comes with a native Divider view, not sure if it fits for you in these cases